src/rmfix/cli.ts

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bun
+import { program } from 'commander';
+import { runRmfix } from './rmfix';
+import type { RmfixCoreOptions, RmfixCliOptions } from './types';
+import { setDebug, setQuiet } from '../rmfilter/utils';
+
+program
+  .name('rmfix')
+  .description(
+    'Run a command, capture its output, and if it fails, use rmfilter to help fix the issue.'
+  )
+  .option('--debug', 'Enable debug logging for rmfix itself')
+  .option('--quiet', 'Suppress rmfix informational logging (not the command output)')
+  .option('--format <type>', 'Specify output format (json, tap, text, auto)', 'auto')
+  .arguments('<command_with_args...>')
+  .action(async (commandWithArgs: string[]) => {
+    const rmfixOwnOptions = program.opts();
+    let commandToRunAndItsArgs: string[];
+    let rmfilterArgs: string[] = [];
+
+    const separatorIndex = commandWithArgs.findIndex((arg) => arg === '--' || arg === '///');
```

Comment:
This should look for a triple dash first and if that exists use it as the separator. Otherwise use the double dash index.